src/elisp/treemacs-customization.el

```diff
@@ -209,7 +209,7 @@ To keep the alist clean changes should not be made directly, but with
   :type 'regexp
   :group 'treemacs)
 
-(defcustom treemacs-sorting 'alphabetic-desc
+(defcustom treemacs-sorting 'alphabetic-asc
```

Comment:
Installation snippet in the readme should be updated as well.

Reply:
Done.